code/modules/mob/living/carbon/xenomorph/castes/gorger/abilities_gorger.dm

```diff
@@ -380,7 +380,7 @@
 	name = "Feast"
 	action_icon_state = "feast"
 	mechanics_text = "Enter a state of rejuvenation. During this time you use a small amount of blood and heal. You can cancel this early."
-	cooldown_timer = 180 SECONDS
+	cooldown_timer = 30 SECONDS
```

Comment:
```suggestion
	cooldown_timer = 100 SECONDS
```
You're lieterally reducing this by like, 3 minutes. Try 100 seconds. First. Reduce later.